packages/boot/test/bootstrapTests/ec-membership-update.test.ts

```diff
@@ -401,7 +411,262 @@ test.serial(
     await advanceTimeBy(1, 'minutes');
 
     const lastOutcome = await governanceDriver.getLatestOutcome();
-    t.notDeepEqual(getUpdatedDebtLimit(), 500_000_000n);
+    const managerParams = getVstorageData(managerGovernanceKey);
+    t.notDeepEqual(managerParams.current.DebtLimit.value.value, 500_000_000n);
     t.assert(lastOutcome.outcome === 'fail');
   },
 );
+
+test.serial('successful proposal by outgoing member', async t => {
+  // Ability to propose by outgoing member should still exist
+  const { storage, advanceTimeBy, getVstorageData, governanceDriver } =
+    t.context;
+  const newCommittee = governanceDriver.ecMembers.slice(0, 3);
+  const outgoingMember = governanceDriver.ecMembers[3];
+
+  const agoricNamesRemotes = makeAgoricNamesRemotesFromFakeStorage(storage);
+  const { VaultFactory } = agoricNamesRemotes.instance;
+  const { ATOM: collateralBrand, IST: debtBrand } = agoricNamesRemotes.brand;
+
+  t.log('Proposing question using old charter invitation');
+  await governanceDriver.proposeParams(
+    VaultFactory,
+    { DebtLimit: { brand: debtBrand, value: 300_000_000n } },
+    { paramPath: { key: { collateralBrand } } },
+    outgoingMember,
+    getQuestionId(3),
+    offerIds.propose.outgoing,
+  );
+
+  t.like(outgoingMember.getLatestUpdateRecord(), {
+    status: { id: getQuestionId(3), numWantsSatisfied: 1 },
+  });
+
+  t.log('Voting on question using first 2 wallets');
+  await governanceDriver.enactLatestProposal(
+    newCommittee.slice(0, 2),
+    getVoteId(3),
+    offerIds.vote.incoming,
+  );
+  for (const w of newCommittee.slice(0, 2)) {
+    t.like(w.getLatestUpdateRecord(), {
+      status: { id: getVoteId(3), numWantsSatisfied: 1 },
+    });
+  }
+
+  t.log('Waiting for period to end');
+  await advanceTimeBy(1, 'minutes');
+
+  t.log('Verifying outcome');
+  const lastOutcome = await governanceDriver.getLatestOutcome();
+  const managerParams = getVstorageData(managerGovernanceKey);
+  t.deepEqual(managerParams.current.DebtLimit.value.value, 300_000_000n);
+  t.assert(lastOutcome.outcome === 'win');
+});
+
+test.serial('successful proposal and vote for auctioneer', async t => {
```

Comment:
I think of this more as...

```suggestion
test.serial('EC can govern auctioneer parameter', async t => {
```